modules/installation-initializing.adoc

```diff
@@ -112,6 +112,17 @@ and compute nodes.
 sub-domains of this base and will also include the cluster name.
 endif::osp[]
 ... Enter a descriptive name for your cluster.
+ifdef::azure[]
++
+[IMPORTANT]
+====
+All Azure resources that are available through public endpoints are subject to
+resource name restrictions, and you cannot create resources that use certain
```

Comment:
Can you remove the comma? I think it would read better without.
s/restrictions, and /restrictions and 

I like Azure's wording: _cannot use reserved words or trademarks in the name_.

All Azure resources that are available through public endpoints cannot use reserved words or trademarks in the name.

Reply:
No, removing the comma's grammatically incorrect.

I am wary of removing the user from the note. I think Azure's working makes more sense in their context: presented after the user's encountered the error and in a place where it makes sense to imply that all of the trademarks are Microsoft trademarks.